### PR TITLE
fix some compiler warnings

### PR DESCRIPTION
**Pull request by zingale**

### CODE REVIEW

**Comment by zingale** [comment]
all tests pass: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2020-08-15/index.html

File: networks/aprox13/actual_network.H [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#ifndef _actual_network_H_
+ifndef _actual_network_H_
```
Comment (maxpkatz) [inline networks/aprox13/actual_network.H:1]: Looks like a typo here

Comment (zingale) [reply networks/aprox13/actual_network.H]: fixed

**Comment by zingale** [comment]
all tests passed: http://groot.astro.sunysb.edu/Microphysics/test-suite/gfortran/2020-08-23/index.html

**Pull request merged**